src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  let total = days * dayCost;
+  const basicDiscount = 20;
+  const maxDiscount = 50;
+
+  if (days > 2 && days < 7) {
+    total = total - basicDiscount;
+  }
+
+  if (days >= 7) {
+    total = total - maxDiscount;
+  }
```

Comment:
You should return value right after you've found the right condition. Also there's no need to use such a complex condition like `days > 2 && days < 7`
```suggestion
  if (days >= 7) {
    return total - maxDiscount;
  }

  if (days >= 3) {
    return total - basicDiscount;
  }
  
  return total;
```

Reply:
I agree with Illia 👍

Reply:
done!